example/pom.xml

```diff
@@ -40,6 +40,11 @@
 			<artifactId>spring-boot-devtools</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-tomcat</artifactId>
+			<scope>provided</scope>

```

Comment:
Is the resulting WAR intended to be deployed on Tomcat?

Reply:
You can choose you can either run it like a "classical" fat jar
` $java -jar gwt-spring-boot-starter-example-1.0-SNAPSHOT.war`

Or you can choose to deploy it in an already installed tomcat

Both should work just fine

Reply:
Are you sure? Setting the scope to "provided" would IMHO lead to the exclusion of the embedded Tomcat so running it via `java -jar` would not work..?

Reply:
I just followed spring doc ;)
https://docs.spring.io/spring-boot/docs/2.1.12.RELEASE/reference/html/build-tool-plugins-maven-plugin.html#build-tool-plugins-maven-packaging

The reason one need to use provided is explain in this other part of spring doc: https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#packaging.layers

> Similarly, an executable war contains the application’s classes in WEB-INF/classes and dependencies in WEB-INF/lib and WEB-INF/lib-provided.

If you run this command line:
`java -jar ....war` spring will use lib and lib-prodived (tomcat lib) to run an embedded container
if you deploy the war in a tomcat, tomcat will only pick /lib (not lib-provided) avoiding a conflict between the tomcat lib in lib-provided and the tomcat installation.

This is also explain here:
> WAR: executable WAR layout. provided dependencies are placed in WEB-INF/lib-provided to avoid any clash when the war is deployed in a servlet container.
https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#packaging

Reply:
You're absolutely right, I missed that! Testing the PR now